src/Native/pkg/runtime.native.System.Data.SqlClient.sni/runtime.native.System.Data.SqlClient.sni.pkgproj

```diff
@@ -10,14 +10,14 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <RuntimeDependency Include="runtime.win7-amd64.runtime.native.System.Data.SqlClient.sni">
+    <Dependency Include="runtime.win7-amd64.runtime.native.System.Data.SqlClient.sni">
       <TargetRuntime>win7-amd64</TargetRuntime>
```

Comment:
After doing so, can you share with me the nuspec you get when building this project as a gist.